src/haptics.cpp

```diff
@@ -12,6 +12,41 @@ init_haptics() {
   }
 }
 
+/* Vibrates specified motors for given frequency.
+ *
+ * @param motor_pin	    specified motor to vibrate
+ * @param duration	    length time of vibration in ms
+ * @param percent_motor percent of haptic vibration capacity to vibrate at,
+ * 			                not to exceed 0.85
```

Comment:
Since this 0.85 is a hard stop that we impose, what are your thoughts on allowing input from 0 to 1.00, but internally this function maps that input to the true 0 to 85%?

Reply:
Good idea